src/ipfs-on-path/scripts/uninstall.js

```diff
@@ -1,5 +1,44 @@
-const { DEST_SCRIPT } = require('./consts')
-const { revert } = require('./backup')
 const { argv } = require('yargs')
+const fs = require('fs-extra')
+const { join } = require('path')
 
-revert(argv.data, 'ipfs', DEST_SCRIPT)
+function checkErr (err) {
+  // NOTE: there is a bug where copySync will throw if the inode
+  // from the src and dst are equal. In the case of symlinks, that'll happen.
+  // https://github.com/jprichardson/node-fs-extra/issues/657
+  if (!err.toString().includes('Source and destination must not be the same')) {
+    throw err
+  }
+}
+
+function revert (userData, bin, dst) {
+  const backup = join(userData, bin + '.bak')
+
+  try {
+    fs.removeSync(dst)
+  } catch (err) {
+    if (err.code !== 'ENOENT') {
+      throw err
+    }
+  }
+
+  try {
+    fs.copySync(backup, dst)
+    console.log(`${backup} restored to ${dst}`)
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      console.log(`${backup} not found, skipping`)
+      return
+    }
+
+    checkErr(err)
+  }
```

Comment:
This _feels_ like a footgun :see_no_evil: 
My thinking: it is possible that someone enabled IPFS on PATH long time ago, and the `ipfs.bak` binary is really, really  old. If that is the case for some of our users, then restoring this ancient binary would create a whole new class of bugs (incompatible repo and/or rpc api).

I think it is better to remove backup but don't touch `/usr/local/bin/ipfs` (if it is a symlink, let it be – lesser evil):

```suggestion
```

Reply:
Okay, I agree with you on this one. I'll merge this bit and so it'll only remove the backup and keep symlinked to IPFS Desktop's binary, which may cause the least damage.

Shouldn't we also remove the symlink too though? Because we say the feature was disabled, so it's a bit odd that we'd keep it.